src/ekobox/gen_op.py

```diff
@@ -0,0 +1,74 @@
+import io
+
+import yaml
+from banana.data import sql
+from ekomark.data import operators
+
+
+def gen_op_card(Q2grid, update=None):
```

Comment:
In operators.py (in ekomark) there is the default op.card I used. Anyway I thought that, to make the user life easier, it would be good to have a function to build an op. card from default (with just the Q2grid  mandatory input) and to modify it in a simple way. So what do you think?